src/controller.coffee

```diff
@@ -214,6 +214,8 @@ exports.Editor = class Editor
     # ## Tracker Events
     # We allow binding to the tracker element.
     dispatchMouseEvent = (event) =>
+      # Firefox doens't have a global event object
+      window.event = event
```

Comment:
Googling seems to imply that window.event is actually IE only (https://developer.mozilla.org/en-US/docs/Web/API/Window/event). Would the better fix be to make it so that nobody depends on event being in the global namespace?